drivers/via/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/byuoitav/av-control-api/drivers/via
+
+go 1.13
+
+replace github.com/byuoitav/kramer-driver => /home/creeder/go/src/github.com/byuoitav/kramer-driver
```

Comment:
Let's get rid of this replace line, it's currently breaking the test/lint pipelines. The pipeline may still fail until the changes to kramer-driver are merged, but at least we'll be able to re-run those once this is gone.